### PR TITLE
FHIR-46253, FHIR-46162, FHIR-46232, FHIR-46373, FHIR-46207, FHIR-44736, FHIR-40511, FHIR-45140, FHIR-46354

### PR DESCRIPTION
**Pull request by mholck**
Updated the parameter descriptions for reportType, subject, and the return

### CODE REVIEW

**Review by yanheras — APPROVED** [review]

File: input/resources/StructureDefinition-cqf-measurePopulationMembership.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      </type>
+      <fixedUri value="http://hl7.org/fhir/StructureDefinition/cqf-measurePopulationMembership"/>
+    </element>
+    <element id="Extension.value[x]">
+      <path value="Extension.value[x]"/>
+      <short value="The number of subjects in the measure population after exclusions."/>
+      <min value="0"/>
+      <max value="1"/>
+      <type>
+        <code value="integer"/>
```
Comment (yanheras) [inline input/resources/StructureDefinition-cqf-measurePopulationMembership.xml:58]: The https://jira.hl7.org/browse/FHIR-44736 resolution says "0..1 Quantity". Change integer to Quantity.

File: input/resources/StructureDefinition-cqf-denominatorMembership.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      </type>
+      <fixedUri value="http://hl7.org/fhir/StructureDefinition/cqf-denominatorMembership"/>
+    </element>
+    <element id="Extension.value[x]">
+      <path value="Extension.value[x]"/>
+      <short value="The number of subjects in the denominator after exceptions and exclusions."/>
+      <min value="0"/>
+      <max value="1"/>
+      <type>
+        <code value="integer"/>
```
Comment (yanheras) [inline input/resources/StructureDefinition-cqf-denominatorMembership.xml:58]: The https://jira.hl7.org/browse/FHIR-44736 resolution says "0..1 Quantity". Change integer to Quantity.

File: input/resources/StructureDefinition-cqf-numeratorMembership.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      </type>
+      <fixedUri value="http://hl7.org/fhir/StructureDefinition/cqf-numeratorMembership"/>
+    </element>
+    <element id="Extension.value[x]">
+      <path value="Extension.value[x]"/>
+      <short value="The number of subjects in the numerator after exceptions and exclusions."/>
+      <min value="0"/>
+      <max value="1"/>
+      <type>
+        <code value="integer"/>
```
Comment (yanheras) [inline input/resources/StructureDefinition-cqf-numeratorMembership.xml:58]: The https://jira.hl7.org/browse/FHIR-44736 resolution says "0..1 Quantity". Change integer to Quantity.

File: input/resources/OperationDefinition-deqm.evaluate-measure.xml [hunk 4]
```diff
@@ -91,11 +91,11 @@
         <searchType value="reference"/>
     </parameter>
     <parameter>
-        <name value="practitioner"/>
+        <name value="provider"/>
         <use value="in"/>
         <min value="0"/>
         <max value="1"/>
-        <documentation value="Practitioner (references [QICore Practitioner](http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-practitioner)) for which the measure will be calculated. If specified, the measure will be calculated only for subjects that have a primary relationship to the identified practitioner"/>
+        <documentation value="The provider for which the report will be run. This may be a reference to a Practitioner, PractitionerRole, or Organization. If specified, the measure will be calculated for subjects that have a primary relationship to the identified provider. How this relationship is determined is implementation-specific. If specified, the measure will be calculated only for subjects that have a primary relationship to the identified provider"/>
```
Comment (yanheras) [inline input/resources/OperationDefinition-deqm.evaluate-measure.xml:98]: Remove the last sentence "If specified, the measure will be calculated only for subjects that have a primary relationship to the identified provider". It is redundant.

**Review by yanheras — CHANGES_REQUESTED** [review]

File: input/examples/MeasureReport-summ-attestation.json [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  "id": "summ-attestation",
+  "meta": {
+      "profile": [
+          "http://hl7.org/fhir/us/davinci-deqm/StructureDefinition/summary-measurereport-deqm"
+      ],
+      "source": "http://example.org/fhir/server"
+  },
+  "status": "complete",
+  "type": "summary",
+  "measure": "http://example.org/fhir/Measure/CQFMAttestationMeasure",
```
[the remark this answers is not in the document]
Comment (mholck) [reply input/examples/MeasureReport-summ-attestation.json:12]: Changed measure name as requested

File: input/resources/OperationDefinition-deqm.evaluate-measure.xml (hunk 4 above)
Comment (mholck) [reply input/resources/OperationDefinition-deqm.evaluate-measure.xml]: removed sentence as requested

File: input/resources/OperationDefinition-deqm.submit-data.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  "system": false,
+  "type": true,
+  "instance": false,
+  "parameter": [
+    {
+      "name": "bundle",
+      "use": "in",
+      "min": 1,
+      "max": "*",
+      "documentation": "The Bundles that contain one or more MeasureReports and data of interest. See <section created from FHIR-46372> for discussion about the Bundles.",
```
[the remark this answers is not in the document]
Comment (mholck) [reply input/resources/OperationDefinition-deqm.submit-data.json]: Updated reference to new 2.1.4 section

File: input/resources/StructureDefinition-cqf-denominatorMembership.xml (hunk 2 above)
Comment (mholck) [reply input/resources/StructureDefinition-cqf-denominatorMembership.xml:58]: All other population counts are integer why would we make this a Quantity and therefore require a unit. I saw the disposition but figured it was a mistake. What would the unit be if they have to provide one?

File: input/resources/StructureDefinition-cqf-measurePopulationMembership.xml (hunk 1 above)
Comment (mholck) [reply input/resources/StructureDefinition-cqf-measurePopulationMembership.xml:58]: See above

File: input/resources/StructureDefinition-cqf-numeratorMembership.xml (hunk 3 above)
Comment (mholck) [reply input/resources/StructureDefinition-cqf-numeratorMembership.xml:58]: see above

File: input/resources/StructureDefinition-cqf-measurePopulationMembership.xml (hunk 1 above)
Comment (mholck) [reply input/resources/StructureDefinition-cqf-measurePopulationMembership.xml:58]: Changed to Quantity for now

File: input/resources/StructureDefinition-cqf-denominatorMembership.xml (hunk 2 above)
Comment (mholck) [reply input/resources/StructureDefinition-cqf-denominatorMembership.xml:58]: Changed to Quantity for now

File: input/resources/StructureDefinition-cqf-numeratorMembership.xml (hunk 3 above)
Comment (mholck) [reply input/resources/StructureDefinition-cqf-numeratorMembership.xml:58]: Changed to Quantity for now

**Review by yanheras — APPROVED** [review]

**Pull request merged**